hub/util/exceptions.py

```diff
@@ -733,3 +733,8 @@ def __init__(self, org_id, creds_key):
         super().__init__(
             f"Unable to find managed credentials '{creds_key}' for organization {org_id}."
         )
+
+
+class EmptyTensorInTheDatasetError(Exception):
+    def __init__(self, tensor_name):
+        super().__init__(f"The tensor {tensor_name} is empty.")
```

Comment:
elaborate more on this. Something like, the dataset has an empty tensor {tensor_name}, pytorch dataloader can't be created. Either populate the tensor or pass tensors argument to .pytorch that excludes this tensor